ci/go-test.sh

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+
+run_go_test
```

Comment:
I don't see where `run_go_test` is coming from - is this a pre-emptive code commit, planning for the future?

Reply:
@grahamwhaley done

Reply:
thx!